src/ui/components/SecondaryToolbox/SecondaryToolbox.css

```diff
@@ -36,16 +37,6 @@
   padding: 8px 12px;
 }
 
-.secondary-toolbox-header .panel-buttons button {
-  padding: 8px 12px;
-}
```

Comment:
GitHub isn't doing a good job of showing this, but this rule was a duplicate:
https://github.com/RecordReplay/devtools/blob/78628cf4485a20e22ce4d30fc79240a295aa98fa/src/ui/components/SecondaryToolbox/SecondaryToolbox.css#L35-L41